### PR TITLE
fix: use httpx for usage tracking

### PR DESCRIPTION
**Pull request by sauyon**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -34,6 +34,7 @@ dependencies = [
     "cloudpickle>=2.0.0",
     "deepmerge",
     "fs",
+    "httpx",
```
Comment (aarnphm) [inline pyproject.toml:37]: Let's also remove requests from the deps?

Comment (sauyon) [reply pyproject.toml:37]: We can't in any of these PRs, it's used by everything else.

Comment (aarnphm) [reply pyproject.toml:37]: so are you going to remove this once all of the PRs are merged?

Comment (sauyon) [reply pyproject.toml:37]: Yep, I think the tests PR is not working, so will probably remove it with that one.

**Review by aarnphm — APPROVED** [review]

**Comment by sauyon** [comment]
I think this PR broke CI @aarnphm going to revert.

**Pull request merged**